runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -46,9 +46,9 @@
     protected abstract UUID sendRequest(ManagementClient client) throws TimeoutException;
 
     /**
-     * Select an orchestrator and return a client. Orchestrator's that
-     * are on unresponsive nodes and the affected endpoint by the workflow
-     * will not be selected. The layout might not reflect the state
+     * Select an orchestrator and return a client. Orchestrator
+     * located on the affected by the workflow endpoint or nodes not responding
+     * to pings will not be selected. The layout might not reflect the state
```

Comment:
Consider rephrasing the whole sentence.
" Select an orchestrator that responds to pings and is not on the same node affected by the workflow. "

Reply:
"located on the affected by the workflow" reads a bit unnatural. I also suggest the re-wording.

Reply:
fixed